syntax = "proto3";
package portrait;




import "capacity_admin_sdk/model/capacity_admin/portrait.proto";

/*
ListPortraits请求
*/
message ListPortraitsRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页大小
    */
    int32 pageSize = 2 ;
    /*
    keyword
    */
    string keyword = 3 ;
}

/*
ListPortraits返回
*/
message ListPortraitsResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated capacity_admin.Portrait list = 4 ;
}

/*
ListPortraitsApi返回
*/
message ListPortraitsResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    portrait.ListPortraitsResponse data = 4 ;
}

